cycle/cycle.env

```diff
@@ -30,19 +30,19 @@ GID=73006
 #
 # Base image from lsstsqre.
 #
-lsstsqre=7-stack-lsst_distrib-w_2020_48
-stack=w_2020_48
+lsstsqre=7-stack-lsst_distrib-w_2021_02
+stack=w_2020_02
 #
 # OpenSplice setup
 #
 dds_community_version=6.9.0
 dds_community_build_el6=2.el6
-dds_community_build_el7=8
+dds_community_build_el7=12
 ts_dds_community=6.9.190705.post8
 dds_private_version=6.10.4
 dds_private_build_el6=1
 dds_private_build=6.el7
-ts_dds_private=6.10.4_6
+ts_dds_private=6.10.4
```

Comment:
Did something change in how the package is being versioned?

Reply:
No, I build py38 and it had this tag in it. I need to roll back to `4_6`